docs/self-managed/identity/user-guide/managing-user-access.md

```diff
@@ -0,0 +1,45 @@
+---
+id: managing-user-access
+title: "Managing user access to Identity"
+sidebar_label: "Managing user access"
+---
+
+In this guide, you will learn how to grant users different levels of access to the Identity application.
+
+## Permissions supported by Identity
+
+Identity implements the following permissions:
+
+- `read`: Users can access all pages in Identity. They _cannot_ create, modify, or delete any data.
+- `read:users`: Users can access only the **Users** page and related subpages.
+- `write`: Users have access to all pages. They can create, modify, and delete data.
+
+:::note
+If you are connecting Identity to your existing Keycloak installation, the `read:users` permission needs to be created
+manually in Keycloak.
+:::
+
+You can [assign the above permissions to users as part of a role](assigning-a-role-to-a-user.md).
+This gives the user access to the Identity application.
+
+## Assign Identity permissions to a user
+
+Without an Identity permission, users can access Identity only to log in to other applications.
+They are not able to access the Identity UI beyond the login page.
+
+To grant a user access to the UI, you need to assign at least one Identity permission as part of a role to the user. This can be achieved in one of the following ways:
+
+### Using Keycloak presets
+
+When you use our Keycloak presets, Identity creates the `Identity` role automatically.
+The role contains the necessary permissions to give a user full read and write access to Identity.
+
+When Web Modeler is connected to Identity, Identity creates the `Web Modeler` role. This role contains the Identity
```

Comment:
I think here we should be using consistent terms so instead of connected I would instead say something like "When Web Modeler is initialised in Identity, ..." as the users are told about component initialisation in other guides

Reply:
👍